ci-operator/step-registry/stackrox/automation-flavors/ibmcloudz-e2e/cluster-create/stackrox-automation-flavors-ibmcloudz-e2e-cluster-create-commands.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+cd /tmp
+pwd
+
+# install kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+chmod +x kubectl
+
+SECRET_DIR="/tmp/vault/ibmcloudz-rhr-creds"
+PRIVATE_KEY_FILE="${SECRET_DIR}/PRIVATE_KEY_FILE"
+
+# https://docs.ci.openshift.org/docs/architecture/step-registry/#sharing-data-between-steps
+KUBECONFIG_FILE="${SHARED_DIR}/kubeconfig"
+
+SSH_KEY_PATH="/tmp/id_rsa"
+SSH_ARGS="-i ${SSH_KEY_PATH} -o MACs=hmac-sha2-256 -o StrictHostKeyChecking=no -o LogLevel=ERROR -o UserKnownHostsFile=/dev/null"
+SSH_CMD="/home/ubuntu/stackrox-ci/k8smanager create"
+
+# setup ssh key
+cp -f $PRIVATE_KEY_FILE $SSH_KEY_PATH
+chmod 400 $SSH_KEY_PATH
+
+# create a k8s cluster
+ssh $SSH_ARGS root@163.66.94.115 "$SSH_CMD" > $KUBECONFIG_FILE
```

Comment:
This line is confusing to me: 
* who is `163.66.94.115`?
* I also need to find the `SSH_CMD` definition to understand what happens - creating a K8s cluster is not the first thing I'd associate with issuing a K8s command.

Reply:
hi @tommartensen, the goal here is for ibm z plaftform enablement for stackrox ci, i.e., to create and destroy ibm z K8s cluster on IBM Cloud as the first step. IBM Cloud currently doesn't support k8s z cluster management by default. So we have to remotely log into an intermediate node (163.66.94.115), and perform a installation/destroy through script called `k8smanager`, which is already set up there. This is a very similar process to the IBM power platform enablement: https://github.com/openshift/release/commit/498265a7f38b7bdc0420a9aec0331f438555cfb9